Source/SwiftLintFramework/Extensions/File+Cache.swift

```diff
@@ -88,7 +88,7 @@ extension File {
 
     internal var assertHandler: (() -> ())? {
         get {
-            return assertHandlers[cacheKey] ?? nil
+            return assertHandlers[cacheKey]
```

Comment:
I don't think so, because I've changed the type of the dictionary to be `[String: AssertHandler]` (it was `[String: AssertHandler?]`). We also only set the value in the setter. 

That `??` was to prevent a double optional, which would be weird.